Documentation/gettingstarted/egress-gateway.rst

```diff
@@ -91,8 +91,7 @@ The egress gateway feature and all the requirements can be enabled as follow:
                --set egressGateway.enabled=true \\
                --set bpf.masquerade=true \\
                --set kubeProxyReplacement=strict \\
-               --set l7Proxy=false \\
-               --set rollOutCiliumPods=true
```

Comment:
ack, I thought `rollOutCiliumPods=true` would restart also the operator :disappointed: